server/src/services/promptService.ts

```diff
@@ -0,0 +1,14 @@
+import type { ContractData } from './contractAnalysisService'
+
+export function generateContractPrompt(contractData: ContractData): string {
+  const prefix = `This is a lease contract.
+The following text has been extracted from a contract photo using an OCR service. 
+In addition to raw text I have extracted which checkboxes are checked in the contract. 
```

Comment:
Kysyin chatgpt ton linen et voisko sitä "parantaa" Anto tälläsen:
"In addition to extracting the raw text from the contract, I have also determined which checkboxes are marked."

Reply:
Joo varmasti tätä voi parantaa moninki eri tavoin ja se on mun mielestä seuraavaks just se tärkein asia mitä tää ohjelma vaatii. Tän pullarin tarkotus oli vaan saada jotain alulle mistä lähteä liikkeelle.